### PR TITLE
Rule updates 2019 06.v4

### PR DESCRIPTION
**Pull request by Kaizhe**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flaky-test

> If contributing rules or changes to rules, please make sure to uncomment the appropriate kind

/kind rule/update

> /kind rule/create

**Any specific area of the project related to this PR?**

> /area engine
/area rules
> /area deployment
> /area integrations
> /area examples

**What this PR does / why we need it**:
Improve whitelisting to reduce false positive

**Which issue(s) this PR fixes**:

Fixes #
1. add placeholder for rules write below root/etc
2. add `weaveworks/scope` to whitelist in rule change thread namespace
3. add openshift image to whitelist

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
1. add placeholder for rules write below root/etc
2. add `weaveworks/scope` to whitelist in rule change thread namespace
3. add openshift image to whitelist
```

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -1148,6 +1148,10 @@
 - macro: user_known_write_etc_conditions
   condition: proc.name=confd
 
+# This is a placeholder for user to extend the whitelist for write below etc rule
+- macro: user_know_write_below_etc_activities
```
Comment (mstemm) [inline rules/falco_rules.yaml]: can you make this user_known instead of user_know?

File: rules/falco_rules.yaml [hunk 2]
```diff
@@ -1470,12 +1480,18 @@
 - list: user_known_change_thread_namespace_binaries
   items: []
 
+- macro: user_know_change_thread_namespace_activities
```
Comment (mstemm) [inline rules/falco_rules.yaml]: Same know -> known thing here.

Comment (Kaizhe) [reply rules/falco_rules.yaml]: done. thanks!

File: rules/falco_rules.yaml (hunk 1 above)
Comment (Kaizhe) [reply rules/falco_rules.yaml]: Done Thanks!

**Comment by Kaizhe** [comment]
@mstemm can you please merge it for me?

**Pull request merged**